### PR TITLE
Clean up logging usage

### LINKED ISSUE #369: Remove `vdirsyncer.log.get`
It is replaced by `click_log` and shouldn't be necessary anymore. Also the fact that other modules don't just use `logging.getLogger` was because I misunderstood how the logging module works.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/31724879-remove-vdirsyncer-log-get?utm_campaign=plugin&utm_content=tracker%2F574345&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F574345&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by hobarrera**
Fixes #369

### CODE REVIEW

File: vdirsyncer/cli/__init__.py [hunk 1]
```diff
@@ -38,7 +39,7 @@ def inner(*a, **kw):
 
 
 def validate_verbosity(ctx, param, value):
-    x = getattr(log.logging, value.upper(), None)
+    x = getattr(logging.getLogger, value.upper(), None)
```
Comment (untitaker) [inline vdirsyncer/cli/__init__.py]: Not quite. :)

Comment (untitaker) [inline vdirsyncer/cli/__init__.py]: concretely, it should be just `logging` since `value` is a log level.

Comment (hobarrera) [inline vdirsyncer/cli/__init__.py]: It looks like this function is never called though, so is it safe to delete it? Or did I miss something?

Comment (untitaker) [inline vdirsyncer/cli/__init__.py]: Oh, yeah...!

**Comment by untitaker** [comment]
@hobarrera style build fails, please run `make install-style style` locally to see issues.

**Comment by hobarrera** [comment]
`flake-import-order` seems odd, because it doesn't sort things like `isort` does (which is, alphabetically, and `from...` imports first).

Fixed though.

**Comment by untitaker** [comment]
Always up for discussion. I sometimes get it wrong too.

**Comment by untitaker** [comment]
Thanks!

**Pull request merged**